src/checkisomd5.c

```diff
@@ -24,8 +24,8 @@
 #include <popt.h>
 #include <termios.h>
 
-#include "md5.h"
-#include "libcheckisomd5.h"
+#include "./include/libcheckisomd5.h"
```

Comment:
This shouldn't be needed if you can pass the include path to it from cmake

Reply:
Well do I want to tell cmake to search in include?

Reply:
Yes, telling cmake about ./include/ is the right way to do it.